Python/codecs.c

```diff
@@ -1526,7 +1526,7 @@ _PyCodec_InitRegistry(PyInterpreterState *interp)
     // search functions, so this is done after everything else is initialized.
     PyObject *mod = PyImport_ImportModule("encodings");
     if (mod == NULL) {
-        return PyStatus_Error("Failed to import encodings module");
+        return PyStatus_Error("Failed to import encodings module. Are you sure PYTHONHOME is correct?");
```

Comment:
Can we tell the user what the (wrong) value of `PYTHONHOME` is?

Reply:
We could, but that significantly complicates things, because the error message has to get put on the heap (`PyStatus` can't do that, AFAIK). There's also a chance it's unset in the case of embedding.